src/go/rpk/pkg/cli/cmd/status_test.go

```diff
@@ -148,6 +154,11 @@ func TestStatus(t *testing.T) {
 			if tt.expectedOut != "" {
 				require.Regexp(t, tt.expectedOut, out.String())
 			}
+			if tt.expectNoOut {
+				require.NotRegexp(t, `\s\sVersion`, out.String())
```

Comment:
If the test isn't for an empty output, what does get output? Should it be nothing?

Reply:
I guess a better name for the field would be `expectNoReport`.

Reply:
But what does get output? Anything that's output will interfere with the redpanda logs, if everything goes to stdout,

Reply:
right, wouldn't it be `out.String().isEmpty()` kinda thing

Reply:
Ah, I see what you're saying. During testing, the log level is set at `Debug` (https://github.com/vectorizedio/redpanda/pull/329/commits/3a3f731158189f122d83b056d6cf31997b5eaaa4#diff-b069ff830c81a4621db72377774629efdb02f8c7782a3740b84e6e71a9783fedR135) because there are tests that check some debug-level logs. So any debug-level messages will appear there.

IRL, the only output people might run into if they pass `-s` are error-level messages, such as 
`Missing license key. Please get a new one at https://vectorized.io/download-trial/`.

Reply:
Cool. We should also make that error message look less like failure.

Reply:
I will remove it altogether soon.

Reply:
Tracking in https://github.com/vectorizedio/redpanda/issues/335